### PR TITLE
dont attempt to publish commits on forks

### PR DESCRIPTION
**Pull request by UpcraftLP**
---
See preview on Cloudflare Pages: https://preview-211.quiltmc-org.pages.dev

### CODE REVIEW

File: .github/workflows/commit-preview.yaml [hunk 1]
```diff
@@ -51,6 +51,7 @@ jobs:
           retention-days: 1
 
       - name: Publish to Pages
+        if: github.repository == 'QuiltMC/quiltmc.org'
```
Comment (Pyrofab) [inline .github/workflows/commit-preview.yaml:54]: shouldn't it check for existence of the token instead ?

Comment (UpcraftLP) [reply .github/workflows/commit-preview.yaml:54]: Secrets cannot be referenced directly, and I did not want to first store it in an environment variable or previous step output.
see https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-secrets

Comment (Pyrofab) [reply .github/workflows/commit-preview.yaml:54]: What's the issue with storing it in an environment variable ? It seems like the recommended way

Comment (Pyrofab) [reply .github/workflows/commit-preview.yaml:54]: I guess if we did that we should ideally split the last two steps to a separate job though

Comment (UpcraftLP) [reply .github/workflows/commit-preview.yaml:54]: Possibly, yeah.

**Review by Pyrofab — APPROVED** [review]
LGTM

**Pull request merged**